samples/net/dns_resolve/README.rst

```diff
@@ -122,9 +106,15 @@ Open a terminal window and type:
 
     $ killall -s KILL dnsmasq
 
-
 Try to launch the dnsmasq application again.
 
+For testing mDNS, use Avahi script in net-tools project:
```

Comment:
For clarity,

     For testing mDNS, use the Avahi script, also found in the GitHub
     zephyrproject-rtos/net-tools project area: